addons/kotsadm/alpha/install.sh

```diff
@@ -34,6 +34,10 @@ function kotsadm() {
         cp "$DIR/addons/kotsadm/alpha/kotsadm-airgap.yaml" "$DIR/kustomize/kotsadm/kotsadm-airgap.yaml"
         insert_patches_strategic_merge "$DIR/kustomize/kotsadm/kustomization.yaml" kotsadm-airgap.yaml
     fi
+    if [ -n "$INSTALLATION_ID" ]; then
+        cp "$DIR/addons/kotsadm/alpha/kotsadm-installation-id.yaml" "$DIR/kustomize/kotsadm/kotsadm-installation-id.yaml"
```

Comment:
I think you want `render_yaml_file` to fill in the ID https://github.com/replicatedhq/kURL/blob/b02a531491e82d604e8df14d0c673825176c3e05/addons/kotsadm/1.34.0/install.sh#L29